### DISCUSSION TITLE
Why StackSetsResource has been abandoned?

### DISCUSSION THREAD

**Discussion by citrusoft** [aws-cloudformation/aws-cloudformation-templates#401] [General]
Thanks to @ckamps, Chuck Meyer et al for creating, blogging and maintaining [StackSetsResource](https://github.com/awslabs/aws-cloudformation-templates/tree/master/aws/solutions/StackSetsResource).

So, the project depends on crhelper which appears defective due to https://aws.amazon.com/blogs/developer/removing-the-vendored-version-of-requests-from-botocore/

I am trying to figure out whether or not I should repair the defect or has something better supplanted this solution.

It appears the last merge to master was back in Nov 2018.
Has this project been abandoned?
If so, why?  what replaces it?

**Comment by ckamps** [comment]
Hi @citrusoft, 

There's an emerging AWS::CloudFormation::StackSet resource that was developed earlier this year and is going through the official internal integration process.  This is being done in the context of the [AWS CloudFormation CLI + Registry] (https://aws.amazon.com/blogs/aws/cloudformation-update-cli-third-party-resource-support-registry/) support that was introduced late in 2019.

See the [PR](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation/pull/6) and associated [Issue #102](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/102]) for the request to integrate a StackSet resource.  

You can learn more about [Creating Resource Providers](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-types.html). This information should enable you to take the not yet generally released integrated StackSet resource referenced above and try it out in your own AWS account via a private resource provider. See [AWS CloudFormation Registry](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html).

You can gain insight into upcoming resource support via the [AWS CloudFormation Coverage Roadmap](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap).  The official [AWS CloudFormation Release History](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/ReleaseHistory.html) can be monitored to see when resources become officially supported.

**Comment by ckamps** [comment]
@citrusoft -  FYI - AWS CloudFormation now supports a StackSets resource type in the CloudFormation registry:

https://aws.amazon.com/about-aws/whats-new/2020/09/aws-cloudformation-now-supports-stacksets-resource-type-in-the-cloudformation-registry/
